services-custom/s3-transfer-manager/src/main/java/software/amazon/awssdk/transfer/s3/internal/serialization/TransferManagerJsonUnmarshaller.java

```diff
@@ -68,10 +75,38 @@ private Function<String, Instant> safeParseDate(Function<String, Instant> dateUn
         }
     };
 
+    TransferManagerJsonUnmarshaller<Map<String, Object>> MAP = new TransferManagerJsonUnmarshaller<Map<String, Object>>() {
+
+        @Override
+        public Map<String, Object> unmarshall(JsonNode jsonContent, SdkField<?> field) {
+            if (jsonContent == null || jsonContent.isNull()) {
```

Comment:
Query : jsonContent.isNull does this mean empty json {} , why dont we use [NullJsonNode](https://github.com/aws/aws-sdk-java-v2/blob/master/core/json-utils/src/main/java/software/amazon/awssdk/protocols/jsoncore/internal/NullJsonNode.java)

Reply:
What do you mean? Looks like `jsonContent.isNull` return true it's an instance of NullJsonNode

Reply:
Query : I meant  returning null when jsonContent == null looks reasonable, however when we have a empty json like `{}` why do not we return an empty map  ?
` return  new HashMap<>();`

Reply:
Ah I see. Yeah, I think that's fair. Will update